openhtf/io/output/mfg_inspector.py

```diff
@@ -348,7 +348,7 @@ def from_json(cls, json_data):
                keydata=json_data['private_key'],
                token_uri=json_data['token_uri'])
 
-  def UploadTestRun(testrun, destination, credentials=None):
+  def UploadTestRun(self, testrun, destination, credentials=None):
```

Comment:
ha, this is a silly bug, sorry about that!  Since self isn't actually used anywhere, can you decorate this with `@staticmethod` instead of adding the self argument?  Thanks!